backend/prompt_studio/prompt_studio_output_manager_v2/serializers.py

```diff
@@ -25,4 +26,33 @@ def to_representation(self, instance):
             )
             token_usage = {}
         data["token_usage"] = token_usage
+        # Get the coverage for the current tool_id and profile_manager_id
+        try:
+            # Fetch all relevant outputs for the current tool and profile
+            prompt_outputs = (
+                PromptStudioOutputManager.objects.filter(
+                    tool_id=instance.tool_id,
+                    profile_manager_id=instance.profile_manager_id,
+                    prompt_id=instance.prompt_id,
+                )
+                .values("prompt_id", "profile_manager_id")
+                .annotate(document_count=Count("document_manager_id"))
+            )
+
+            coverage = {}
+            for prompt_output in prompt_outputs:
+                prompt_key = str(prompt_output["prompt_id"])
+                profile_key = str(prompt_output["profile_manager_id"])
+                coverage[f"coverage_{prompt_key}_{profile_key}"] = prompt_output[
+                    "document_count"
+                ]
+
+            data["coverage"] = coverage
+        except Exception as e:
+            logger.error(
```

Comment:
@jagadeeswaran-zipstack either make this a warning log or mention that we'll be continuing processing / suppressing the error in the log message

Reply:
Got it

Reply:
got it